docs/UserGuide.md

```diff
@@ -265,13 +310,23 @@ the data of your previous StoreMando home folder.
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Command summary
+## 5. Command summary <a name = "commandsummary"></a>
```

Comment:
There will be a need to look into this section to double-check, examples will be reminder and sort. :)